fixtures/parallelizer/__init__.py

```diff
@@ -215,6 +216,25 @@ def __init__(self, config):
         else:
             # Using sprout
             self.sprout_client = SproutClient.from_config()
+            try:
+                if self.config.option.sprout_desc is not None:
+                    jenkins_job = re.findall(r"Jenkins.*[^\d+$]", self.config.option.sprout_desc)
```

Comment:
@psav in jenkins-runner.sh the pool desc is set as "--sprout-desc 'Jenkins tag: ${BUILD_TAG}'"
the BUILD_TAG is jenkins job+build number. so In the above regex I'm trying to search job desc starting with Jenkins and ending with digits.

sample output:
Loading templates from trackerbot
no test annotation found in None
Check if pool already exists for this 'Jenkins tag: jenkins-perf-test-downstream-55z-workload-' Jenkins job
Destroying the old pool 29830 for 'Jenkins tag: jenkins-perf-test-downstream-55z-workload-' job.